README.md

```diff
@@ -113,6 +113,22 @@ The Criteria Section of the config file is simply named "Criteria" (without the
 - 	Filters: a list of the names of the filters you want to apply
 - 	Start Date: a date filter in the format YYYY-MM-DD which will exclude issues with resolved dates before the provided date (optional)
 - 	End Date: a date filter in the format YYYY-MM-DD which will include issues with resolved dates before the provided date (optional)
+-   BlockedAttributes: This optional feature will add and value the ‘Days Blocked’ column to the CSV extract with the number of days blocked for each issue. 
+    -   The number of days blocked is calculated by using the workflow function to block and unblock an issue.
+		Days Blocked is calculated down to the tenth of a day.  For example, the Days Blocked might calculate to 15.4.
+    -   Number of blocked days  = the total of all blocked days across all status values included in the filter.
+    -   Multiple block and unblocks are allowed.
+    -   When enabled, the block values are used in the ScatterPlot, Cycle Time Histogram, and Flow Efficiency.
+        o   Cycle Time:
+               issues are displayed as a red dot instead of a black dot. 
```

Comment:
This is not proper markdown and does not render correctly.

Reply:
I'll work on fixing it and providing the correct markdown.

Reply:
Cool, thanks! Not a super big deal, but thought I'd call it out.